pkgs/top-level/all-packages.nix

```diff
@@ -14864,7 +14864,12 @@ with pkgs;
     pulseSupport = config.pulseaudio or true;
   };
 
-  deadbeef-mpris2-plugin = callPackage ../applications/audio/deadbeef/plugins/mpris2.nix { };
+  deadbeefPlugins = {
+    mpris2 = callPackage ../applications/audio/deadbeef/plugins/mpris2.nix { };
+    opus = callPackage ../applications/audio/deadbeef/plugins/opus.nix { };
+  };
+
+  deadbeef-mpris2-plugin = deadbeefPlugins.mpris2; # renamed on 2018-02-23
```

Comment:
Maybe `pkgs/top-level/aliases.nix` instead? Or do we want this permanent?

Reply:
Cool, this will be useful for https://github.com/NixOS/nixpkgs/issues/32804